insights/mappers/systemctl_show.py

```diff
@@ -23,10 +74,19 @@ def parse_content(self, content):
         We will also suppressed empty properties in return data.
         """
         data = {}
+        data = split_kv_pairs(content, use_partition=False, ordered=True)
+
+        """Remove empty key"""
+        self.data = {k: v for k, v in data.iteritems() if not v == ''}
```

Comment:
You could also look at looping through the values and using `.pop()` to remove empty values on the original `data`.